manim/renderer/stream_renderer.py

```diff
@@ -0,0 +1,17 @@
+from .cairo_renderer import CairoRenderer
+from .. import config
+from ..scene.stream_file_writer import StreamFileWriter
+
+
+class StreamCairoRenderer(CairoRenderer):
+    """I wish there was another way to have a renderer that uses the
```

Comment:
Nitpick: can be added as further comments to this PR, should probably be removed before merge.